x/nft/keeper/msg_iris.go

```diff
@@ -0,0 +1,74 @@
+package keeper
```

Comment:
This `nibirutype.msg`  to `iristype.msg` conversion functions are for
commonizing `keeper` and `msg.server` processes